.github/workflows/build.yaml

```diff
@@ -166,10 +166,11 @@ jobs:
           toolchain: ${{ matrix.rust }}
           override: true
       - uses: r7kamura/rust-problem-matchers@v1
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --release --locked --features "sim alloc"
+      - name: Run tests
+        # For some reason `SGX_AESM_ADDR` is set to 1 in the github runners.
+        # Not seeing it documented here, https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md
+        # We can't just set to 0 or "" as the SGX code looks for it being set, not what the value is :(
+        run: unset SGX_AESM_ADDR && cargo test --release --locked --features "sim alloc"
```

Comment:
I'm not fond of having to unset prior to the test invocation.
I removed the installation of SGX and dcap libraries and `SGX_AESM_ADDR` was still set in the workspace.
So far I've been unable to find a command/tool that will persist unsetting of an env variable across steps.

Reply:
resolving since this has been reviewed with no major gripes